provisioning/transport/http/devdoc/http_requirements.md

```diff
@@ -0,0 +1,89 @@
+# azure-iot-provisioning-http Requirements
+
+## Overview
+This module provides HTTP protocol support to communicate with the Azure device provisioning service
+
+```js
+'use strict';
+
+export class Http extends EventEmitter implements DeviceProvisioningTransport {
+  api-version: string;
+  operationPollingIntervalInSeconds: Number
```

Comment:
>operationPollingIntervalInSeconds [](start = 2, length = 33)

intervals are usually in milliseconds in Javascript, not sure what to do here.
Also, 'operation' is not super descriptive of what we're actually polling for.
Last but not least, what are other SDKs doing for APIs to set this?

Reply:
I will compare to other SDKs shortly

---
In reply to: [140062992](https://github.com/Azure/azure-iot-sdk-node/pull/86#discussion_r140062992) [](ancestors = 140062992)

Reply:
I suspect there will be a setOptions thing somewhere :-/

---
In reply to: [140107581](https://github.com/Azure/azure-iot-sdk-node/pull/86#discussion_r140107581) [](ancestors = 140107581,140062992)